crates/web5/src/apid/dids/methods/did_dht/mod.rs

```diff
@@ -28,19 +28,8 @@ pub mod document_packet;
 const JSON_WEB_KEY: &str = "JsonWebKey";
 const DEFAULT_RELAY: &str = "https://diddht.tbddev.org";
 
-fn extract_ed25519_public_key(jwk: &Jwk) -> Result<Vec<u8>> {
-    let decoded_x = general_purpose::URL_SAFE_NO_PAD.decode(&jwk.x)?;
-    if decoded_x.len() != PUBLIC_KEY_LENGTH {
-        return Err(MethodError::DidCreationFailure(format!(
-            "Identity key JWK public key must have length {}",
```

Comment:
I removed this in favor of `ed25519::extract_public_key`. We lose out on the slight difference in error handling, but I think it's fine in service of DRY.